docs/wallet/webhooks.md

```diff
@@ -0,0 +1,55 @@
+---
+id: webhooks
+title: Webhooks
+---
+
+The web wallet webhooks allow you to build or setup integrations for dapps or payment flows.
+
+The web wallet webhooks are links that point the user to the wallet to either login or populate a send transaction form with the provided arguments. Once the action is performed the user is redirected to the provided callback URL along with a success or error status.
```

Comment:
Small readability improvement: `the user to the wallet` => `the user of the wallet`
Please add a comma: `Once the action is performed, the user`

Reply:
thanks for your input, I've added the changes here -- https://github.com/ElrondNetwork/docs.elrond.com/pull/114